frontend/src/views/ElectedCanteens.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div class="text-left">
+    <BreadcrumbsNav />
+    <h1 class="text-h6 mt-n2 mb-2 font-weight-bold">
+      Les cantines de mon territoire
+    </h1>
+    <p>Établissements dans {{ departmentsString }}</p>
+    <v-data-table
+      :footer-props="footerProps"
+      :items="visibleCanteens"
+      :options.sync="options"
+      :loading="loading"
+      :headers="headers"
+      :server-items-length="canteenCount || 0"
+    >
+      <template v-slot:[`item.publicationStatus`]="{ item }">
+        <router-link
+          v-if="item.publicationStatus === 'published'"
+          :to="{ name: 'CanteenPage', params: { canteenUrlComponent: `${item.id}--${item.name}` } }"
+        >
+          Publiée
+        </router-link>
+        <span v-else>
+          Non publiée
+        </span>
+      </template>
+    </v-data-table>
+  </div>
+</template>
+
+<script>
+import jsonDepartments from "@/departments.json"
+import { getObjectDiff } from "@/utils"
+import BreadcrumbsNav from "@/components/BreadcrumbsNav"
+
+export default {
+  name: "ElectedCanteens",
+  components: { BreadcrumbsNav },
+  data() {
+    return {
+      loading: false,
+      canteenCount: null,
+      visibleCanteens: [],
+      limit: 10,
+      options: {
+        sortBy: [],
+        sortDesc: [],
+        page: 1,
+      },
+      footerProps: {
+        disableItemsPerPage: true,
+        itemsPerPageText: "Cantines per page",
+      },
+      headers: [
+        { text: "SIRET", value: "siret", sortable: true },
+        { text: "Nom", value: "name", sortable: true },
+        { text: "Ville", value: "city", sortable: true },
+        { text: "Couverts moyen par jour", value: "dailyMealCount", sortable: true },
+        { text: "Publication", value: "publicationStatus", sortable: false },
+      ],
+    }
+  },
+  computed: {
+    offset() {
+      return (this.options.page - 1) * this.limit
+    },
+    departmentsString() {
+      const userDepartments = this.$store.state.loggedUser.departments
+      const departmentsArray = jsonDepartments
+        .filter((x) => userDepartments.indexOf(x.departmentCode) > -1)
+        .map((x) => `${x.departmentName} (${x.departmentCode})`)
+      if (departmentsArray.length < 2) return `le département ${departmentsArray.join("")}`
+      return `les départements ${departmentsArray.slice(0, -1).join(", ")} et ${departmentsArray.slice(-1)}`
```

Comment:
If the person is listed as an elu but they aren't given any departments due to user error, it might be worth showing a different kind of message?

Reply:
Changement ici https://github.com/betagouv/ma-cantine/pull/2904/commits/949aa0627e65193afb5037839c13c4570ef1a2ae
![image](https://github.com/betagouv/ma-cantine/assets/1225929/fbc7caeb-1a05-4ef4-a37c-6876f3355626)
J'ai volontairement laissé la table vide pour laisser comprendre ce que la personne trouvera une fois que les departements seront assignés.